src/main/java/io/github/thebusybiscuit/slimefun4/implementation/accessibility/PlayerAccessDataImpl.java

```diff
@@ -0,0 +1,169 @@
+package io.github.thebusybiscuit.slimefun4.implementation.accessibility;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonPrimitive;
+import io.github.thebusybiscuit.slimefun4.api.accessibility.AccessData;
+import io.github.thebusybiscuit.slimefun4.api.accessibility.AccessLevel;
+import io.github.thebusybiscuit.slimefun4.api.accessibility.ConcreteAccessLevel;
+import io.github.thebusybiscuit.slimefun4.api.accessibility.PlayerAccessData;
+import io.github.thebusybiscuit.slimefun4.core.services.JsonDeserializationService;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import me.mrCookieSlime.Slimefun.api.Slimefun;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import java.util.logging.Level;
+
+/**
+ * Implementation for {@link PlayerAccessData}.
+ */
+public class PlayerAccessDataImpl implements PlayerAccessData {
+
+    static {
+        SlimefunPlugin.getJsonDeserializationService()
+            .registerDeserialization(PlayerAccessDataImpl.class, PlayerAccessDataImpl::new);
+    }
+
+    private final Map<UUID, AccessLevel> levelMap = new HashMap<>();
+
+    public PlayerAccessDataImpl() {
+
+    }
```

Comment:
Again redundant constructor.

Reply:
I don't see why this is redundant. The object itself is more of data wrapper, the overloaded constructors are meant for when you want to initialize with serialized data

Reply:
Ah nevermind, I didn't see the overloaded ones.